### DISCUSSION TITLE
IntelliJ IDEA 2022.3.2 (Ultimate Edition) WSL2 weird behaviours

### DISCUSSION THREAD

**Discussion by meza** [community/community#46561] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Using the new WSL2 setup of intellij, the plugin is acting strange.

One of the biggest issues is that pressing TAB simply doesn't do the completion. I have to open up the plugin window and manually click on accept solution for it to work.
Then most of the times the suggestions don't even show up within the editor itself. This seems to be intermittent behaviour though.

The plugin works perfectly if used on projects that aren't going through the WSL2 integration.

I've tested it with or without the experimental UI. Makes no difference.

Using the github copilot plugin version: 1.2.2.2371

```
IntelliJ IDEA 2022.3.2 (Ultimate Edition)
Build #IU-223.8617.56, built on January 26, 2023
Licensed to ***
You have a perpetual fallback license for this version.
Subscription is active until January 14, 2024.
Runtime version: 17.0.5+1-b653.25 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 24
Registry:
    ide.experimental.ui=true

Non-Bundled Plugins:
    com.jetbrains.plugins.ini4idea (223.8617.59)
    org.intellij.plugins.hcl (223.8214.6)
    com.intellij.plugins.macoskeymap (223.7571.117)
    com.github.copilot (1.2.2.2371)
    com.intellij.ideolog (203.0.30.0)
    org.jetbrains.plugins.go (223.8617.56)
    com.intellij.guice (223.7571.182)
    com.demonwav.minecraft-dev (2022.3-1.5.22)
    com.intellij.lang.jsgraphql (3.3.0)
    com.nekofar.milad.intellij.remix (1.1.6)
    com.dmarcotte.handlebars (223.7571.117)
    mobi.hsz.idea.gitignore (4.4.4)
    net.ashald.envfile (3.4.1)
    ru.adelf.idea.dotenv (2022.3)

Kotlin: 223-1.8.0-release-345-IJ8617.56
```

**Comment by tokenosopher** [comment]
Been having the same problem.

**Comment by meza** [comment]
I've been updating the versions diligently since posting this but the issue still remains 😞
